site/content/kapp-controller/docs/develop/kctrl-upstream-authoring-tutorial.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Packaging upstream artifacts with CLI
+---
+This tutorial explores how `kctrl` allows us to create Carvel packages using existing artifacts like manifests released as a part of a GitHub release or a Helm chart.
```

Comment:
lets add "Available in kctrl v...+" above

Reply:
Will add this ^